rpm/update-config-files.sh

```diff
@@ -17,25 +17,19 @@ ensure() {
     then
```

Comment:
(Pre-existing: it’s idiomatic to use `if grep -q`, rather than redirect to `/dev/null`)